homeassistant/components/ai_task/strings.json

```
@@ -1,8 +1,8 @@
 {
   "services": {
-    "generate_text": {
-      "name": "Generate text",
-      "description": "Use AI to run a task that generates text.",
+    "generate_data": {
+      "name": "Generate data",
+      "description": "Use AI to run a task that generates data.",
```

To keep a descriptive wording this should use third-person singular as the vast majority of action descriptions in HA.

```suggestion
      "description": "Uses AI to run a task that generates data.",
```